### DISCUSSION TITLE
Reed Stops

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#23] [Ideas]
Can you add the choice of whether a stop is a conventional stop or whether it is a reed stop?

Sample stop images should identify reeds in red, couplers in green and common stops in black font colour.

**Comment by larspalo** [comment]
As far as I know, GrandOrgue doesn't enforce such a color scheme, it's totally up to the discretion of the sample set producer how he/she wants to display things.

To make things easier for the sample set producer I've implemented the possibility in GOODF to copy the gui attributes from one GUI element to any other(s) of the same type (either on the same panel or on another). It's done by creating one gui element on a panel of that type in just the way you wish it to display - then you right click on it in the tree. This brings up a dialog where you can select which target panel and which gui elements on that panel you wish to copy to. It's possible to mark selections of elements with holding shift and click or mark multiple but separate elements by holding ctrl and click to add to the selection. When ready click the "Copy to selected" button.

This should make the task of getting the same color scheme for different object types much easier, even though it's not going to happen automatically... Hope that helps!
